MacBry-Maciej-Bryja/SimpleTodoApp1/src/main/java/com/mac/bry/simpleTodo/Application/Application.java

```diff
@@ -0,0 +1,18 @@
+package com.mac.bry.simpleTodo.Application;
+
+import org.apache.log4j.BasicConfigurator;
+
+import com.mac.bry.simpleTodo.appController.MainController;
+
+public class Application {
+
+	public static void main(String[] args) {
+		//for Log4j run
+		BasicConfigurator.configure();
```

Comment:
Zamiast pisać komentarz po co wywołujesz to wystarczy stworzyć klasę Log4jConfigurator i w niej stworzyć statyczną metodę configure, w której wywołasz kod z linii 11. Komentarz wtedy będzie niepotrzebny, ponieważ po nazwie klasy będzie wiadomo po co to jest robione.

Reply:
Poprawione